src/Client.php

```diff
@@ -329,6 +331,19 @@ public function selectDatabase(string $databaseName, array $options = [])
         return new Database($this->manager, $databaseName, $options);
     }
 
+    public function setLogger(LoggerInterface $logger): void
```

Comment:
How would you remove the logger? It should nullable.

At the very least, it must be unregistered from the driver when the client is destroyed.

Reply:
Resolving, as Client no longer implements LoggerAwareInterface.